content/core-features/authentication.textile

```diff
@@ -455,12 +455,11 @@ A revocation request must include one or more target specifiers which define the
 
 By default, supported target specifiers are:
 
-- clientId:<clientId> := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given.
-- revocationKey:<revocationKey> := Matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@.
-
+- clientId := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given, for example: <code>targets: ["clientId:client1@example.com" ]</code>
```

Comment:
I think we need something before the table to explain the format for target specifiers, so by the time the developer gets into the table they know the format would be along the lines of `clientId: <some_string>`, etc.

I also think we could clarify the description here a bit. For example, `clientId` is the key, not the value. We need to make that crystal clear. The example would then provide extra clarification as to what's going on.